Formula/arxiv_latex_cleaner.rb

```diff
@@ -0,0 +1,31 @@
+class ArxivLatexCleaner < Formula
+  include Language::Python::Virtualenv
+
+  desc "Clean LaTeX code to submit to arXiv"
+  homepage "https://github.com/google-research/arxiv-latex-cleaner"
+  url "https://files.pythonhosted.org/packages/d7/82/b32e1991ee2a8fd0db2f02ba8e135430aaeed9507e8008d5e9c8beec8eff/arxiv_latex_cleaner-0.1.27.tar.gz"
+  sha256 "f347fc6082417316247dca97bb76f15ac677eb1f5e65f091013086a30cac4eda"
+  license "Apache-2.0"
+  head "https://github.com/google-research/arxiv-latex-cleaner.git", branch: "main"
+
+  depends_on "python@3.10"
```

Comment:
Don't you need a `pillow` dependency?

Reply:
Yes, added in [e8720b1](https://github.com/Homebrew/homebrew-core/pull/100405/commits/e8720b15eb6c85b3e4b27d0cf361aa989ebb5fdd).